configure.in.in

```diff
@@ -53,7 +53,7 @@ AC_PROG_CXX
 # gnu++0x instead of c++0x so that
 # we do not lose gnu extensions like typeof
 # FIXME: rename to Y2CORE_CXXFLAGS and change in devtools:y2autoconf
-Y2CORE_CFLAGS="-std=gnu++0x"
+Y2CORE_CFLAGS="-std=gnu++0x -Werror"
```

Comment:
it is very bad idea, because it mean that every new version of g++ break yast2-core and force us to fix any newly added warning. This practice is highly discouraged.